k8s/trillian/manifest/config.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: List
```

Comment:
What is the benefit of using the "kind: List" in the resources definitions? I am afraid that without listing the List type explicitly in the Application resource definition, the resource will not be properly removed when the application is removed from a K8s cluster.

Reply:
It makes it possible to use YAML anchors to deduplicate labels. They aren't actually used in this file, but they're used in most of the others, so I've used a List here for consistency. Anchors only work within a document, so using the "---" separator instead of List breaks the anchors.

Reply:
I can't add List to the app resource definition because:
1. the set_ownership.py script crashes (because it doesn't have metadata, see https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/pull/293)
2. the deployer fails with `Error from server (Forbidden): lists is forbidden: User "system:serviceaccount:apptest-2e3bcar0:apptest-2e3bcar0-deployer-sa" cannot list lists at the cluster scope`

The problem seems to be the assumption that the List exists as a resource in Kubernetes. Instead, I think the Kubernetes server discards the List once it has processed all of the contained resources, so it doesn't need to be annotated with ownership info or deleted at the end.

I think I'll just duplicate the labels since it appears the marketplace tooling isn't able to handle List properly.

Reply:
Lists removed.